recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "rubin-env" %}
 {% set version = "10.0.0" %}
-{% set build_number = 3 %}
+{% set build_number = 4 %}
```

Comment:
Don't we have to change the version number for this?

Reply:
That is certainly the cleaner approach. Also for version 9, etc.

Reply:
v9 is a problem because for the older envs those are essentially static and forcing a whole new env version to use for the older pipeline builds might get complicated.

Reply:
We can solve v9 in a different way later . important is to go forward with fixing release docs v10 plus.

Reply:
ie make a v10.1 with graphviz and then make sure that v29 is built using that? Even if it means that it's inconsistent with the weekly it came from?

Reply:
Would be 10.0.1 and just adding a package not used for the compilation. I don't see a problem as 10.0.X are identical apart from floating package version resolutions in conda.  One could argue this is requires 10.1.0 as it is not just a patch and the "interface" changes.